docs/site/Decorators_inject.md

```diff
@@ -145,57 +170,41 @@ console.log(store.locations); // ['San Francisco', 'San Jose']
 
 ### @inject.view
 
-`@inject.view` injects a `ContextView` to track a list of bindings matching a
-filter function or scope/tags.
+`@inject.view` injects a `ContextView` to track a list of bound values matching
+a filter function.
 
 ```ts
-import {inject, Getter} from '@loopback/context';
+import {inject} from '@loopback/context';
 import {DataSource} from '@loopback/repository';
 
 export class DataSourceTracker {
   constructor(
-    // The target type is `Getter` function
-    @inject.view({tags: ['datasource']})
-    private dataSources: Getter<DataSource[]>,
+    @inject.view(bindingTagFilter('datasource'))
+    private dataSources: ContextView<DataSource[]>,
   ) {}
 
   async listDataSources(): Promise<DataSource[]> {
     // Use the Getter function to resolve data source instances
-    return await this.dataSources();
+    return await this.dataSources.values();
   }
 }
 ```
 
-The `@inject.view` decorator can take a `BindingFilter` function in addition to
-`BindingScopeAndTags`. And it can be applied to properties too. For example:
+In the example above, `bindingTagFilter` is a helper function that creates a
```

Comment:
Personally, I like more `filterByTag` than `bindingTagFilter`. Not a big deal, I am just wondering what other team members think about this name? /cc @strongloop/loopback-maintainers

Reply:
Renamed